### ISSUE TITLE
Can not plot rivers

### ISSUE THREAD

**Issue by NMC-DAVE** [SciTools/cartopy#945]
Hi, guys:
    I use the cartopy 0.15.1 in 64bit windows. And I run the following code:
```python
import matplotlib.pyplot as plt
import cartopy.crs as ccrs
import cartopy.feature

rivers = cartopy.feature.NaturalEarthFeature(
    category='physical', name='rivers_lake_centerlines',
    scale='50m', facecolor='none', edgecolor='blue')

fig, ax = plt.subplots(
    nrows=1, ncols=1, subplot_kw={'projection': ccrs.PlateCarree()},
                         figsize=(10,6))
ax.add_feature(rivers, linewidth=1)
ax.set_extent([-65, -45, -40, -17.5])
plt.show()
```

Got errors, so Why?
```pytb
Traceback (most recent call last):
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\backends\backend_qt5
agg.py", line 197, in __draw_idle_agg
    FigureCanvasAgg.draw(self)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\backends\backend_agg
.py", line 464, in draw
    self.figure.draw(self.renderer)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\artist.py", line 63,
 in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\figure.py", line 114
4, in draw
    renderer, self, dsu, self.suppressComposite)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\image.py", line 139,
 in _draw_list_compositing_images
    a.draw(renderer)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\artist.py", line 63,
 in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "C:\Miniconda3\envs\met\lib\site-packages\cartopy\mpl\geoaxes.py", line 3
72, in draw
    inframe=inframe)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\artist.py", line 63,
 in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\axes\_base.py", line
 2426, in draw
    mimage._draw_list_compositing_images(renderer, self, dsu)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\image.py", line 139,
 in _draw_list_compositing_images
    a.draw(renderer)
  File "C:\Miniconda3\envs\met\lib\site-packages\matplotlib\artist.py", line 63,
 in draw_wrapper
    draw(artist, renderer, *args, **kwargs)
  File "C:\Miniconda3\envs\met\lib\site-packages\cartopy\mpl\feature_artist.py",
 line 140, in draw
    for geom in geoms:
  File "C:\Miniconda3\envs\met\lib\site-packages\cartopy\feature.py", line 117,
in <genexpr>
    extent_geom.intersects(geom))
  File "C:\Miniconda3\envs\met\lib\site-packages\shapely\geometry\base.py", line
 707, in intersects
    return bool(self.impl['intersects'](self, other))
  File "C:\Miniconda3\envs\met\lib\site-packages\shapely\predicates.py", line 13
, in __call__
    self._validate(other, stop_prepared=True)
  File "C:\Miniconda3\envs\met\lib\site-packages\shapely\topology.py", line 18,
in _validate
    raise ValueError("Null geometry supports no operations")
ValueError: Null geometry supports no operations
```

**Comment by byersiiasa** [comment]
I have been having the exact same issue. Works fine for `scale=110m` , but for 50 and 10m throws up this error.

PS - where does one find the actual "names" of the NaturalEarth shapefiles to be used in Cartopy as arguments?
I am assuming it comes from the web address string, substituting the hyphen for an underscore and removing the scale, but I'm not sure. Is there a list somewhere?

**Comment by scaine1** [comment]
I also have this issue. plotting rivers_lake_centerlines fails with any resolution other than 110m..
I tried plotting it the twos ways  below but got the same result.

```python
rivers = cc.feature.NaturalEarthFeature(scale='110m', category='physical',
        name='rivers_lake_centerlines', edgecolor=water_color, facecolor='none')
```
and 

```python
shp_path = cc.io.shapereader.natural_earth(
        resolution='50m',
        category='physical',
        name='rivers_lake_centerlines')
shp_contents = cc.io.shapereader.Reader(shp_path)
river_generator = shp_contents.geometries()
river_feature = cc.feature.ShapelyFeature(
        river_generator,
        cc.crs.PlateCarree(),
        edgecolor=water_color,
        facecolor='none')
ax.add_feature(river_feature)
```
Is the problem with the natural earth dataset itself perhaps?

**Comment by QuLogic** [comment]
Should be fixed by #976.
